graphrepresentation.pl

```diff
@@ -1,8 +1,7 @@
 % This file contains out internal representation for graphs.
 % our approach uses a graph-term from.
 
-:- module(graphrepresentation).
-:- include(graphload).
+:- module(graphrepresentation, [graph/4]).
 :- dynamic graph/4.
```

Comment:
We can get rid of this line no?

Reply:
I left that intentionally - we need to export a dynamic predicate otherwise we'll get modification permission errors

Reply:
marking resolved because I think the dynamic statement is needed